wzprof.go

```diff
@@ -22,11 +22,13 @@ import (
 type Profiling struct {
 	wasm []byte
 
+	onlyFunctions     map[string]struct{}
 	filteredFunctions map[string]struct{}
 	symbols           symbolizer
 	stackIterator     func(mod api.Module, def api.FunctionDefinition, wasmsi experimental.StackIterator) experimental.StackIterator
 
-	isGo bool
+	isGo     bool
+	isPython bool
```

Comment:
Can we turn this into an enum so there is no impossible conditions like having both `isGo` and `isPython` set to true?